Libplanet.Net/Consensus/ConsensusReactor.cs

```diff
@@ -100,6 +103,50 @@ private async Task ProcessMessageHandler(Message message)
                     await ReplyMessagePongAsync(message);
                     _consensusContext.HandleMessage(consensusMessage);
                     break;
+                case Ping ping:
+                    await ReplyMessagePongAsync(ping);
+                    break;
+            }
+        }
+
+        private async Task CheckValidatorsLiveness(CancellationToken ctx)
+        {
+            while (!ctx.IsCancellationRequested)
+            {
+                PingPong sendMessage = async peer =>
+                {
+                    try
+                    {
+                        var pong = await _consensusTransport.SendMessageAsync(
+                            peer,
+                            new Ping(),
+                            TimeSpan.FromSeconds(1),
+                            ctx);
+                        return pong is Pong;
+                    }
+                    catch (Exception)
+                    {
+                        return false;
+                    }
+                };
+
+                var tasks = _validatorPeers
+                    .Select(peer => sendMessage(peer))
+                    .ToList();
+                var countOfPong = (await Task.WhenAll(tasks)).Count(x => x);
+
+                var twoThird = _validatorPeers
+                    .Select(x => x.PublicKey)
```

Comment:
It seems that we're using it for `.Count` only. if then, we don't need `.Select()` here.